soundata/datasets/dcase_bioacoustic.py

```diff
@@ -0,0 +1,628 @@
+"""DCASE-BIOACOUSTIC Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    DCASE-BIOACOUSTIC
+    =================
+    
+    #Development set:
+
+    The development set for task 5 of DCASE 2022 "Few-shot Bioacoustic Event Detection" consists of 192 audio files acquired from different bioacoustic sources. The dataset is split into training and validation sets. 
+
+    Multi-class annotations are provided for the training set with positive (POS), negative (NEG) and unkwown (UNK) values for each class. UNK indicates uncertainty about a class. 
+
+    Single-class (class of interest) annotations are provided for the validation set, with events marked as positive (POS) or unkwown (UNK) provided for the class of interest. 
+
+    this version (3):
+    * fixes issues with annotations from HB set
```

Comment:
I think this is not rendering properly in the docs, it is all placed in the same line

Reply:
done